composer.json

```diff
@@ -8,18 +8,16 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.1.3",
-        "fideloper/proxy": "^4.0",
-        "laravel/framework": "5.8.*",
-        "laravel/tinker": "^1.0"
+        "php": "^8.0",
+        "laravel/framework": "^11.0",
+        "laravel/tinker": "^2.9"
     },
     "require-dev": {
-        "beyondcode/laravel-dump-server": "^1.0",
-        "filp/whoops": "^2.0",
-        "fzaninotto/faker": "^1.4",
```

Comment:
Thanks for swapping this out. I see [fzaninotto/faker](https://packagist.org/packages/fzaninotto/faker) was abandoned.